src/frontend/src/app/model/IssueComment.ts

```diff
@@ -1,8 +1,8 @@
-import {UserProfileDto} from "./UserProfileDto";
+import {UserProfileDto} from './UserProfileDto';
 
 export class IssueComment {
   public id: number;
   public owner: UserProfileDto[];
-  public content: string = '';
+  public content: string;
```

Comment:
czemu usunąłeś?

Reply:
nie widzę potrzeby by to było oraz TSLint wykrywa to jako błąd